packages/js/src/server/check-ins-manager/client.ts

```diff
@@ -22,7 +21,7 @@ export class CheckInsClient {
     const response = await this.transport.send({
       method: 'GET',
       headers: this.getHeaders(),
-      endpoint: `${this.BASE_URL}/v2/project_keys/${projectApiKey}`,
+      endpoint: Util.endpoint(this.config.appEndpoint, `v2/project_keys/${projectApiKey}`),
```

Comment:
`Util.endpoint` is a helper method that provides safer URL generation.
If `this.config.appEndpoint` is _https://app.honeybadger.io/_ or  _https://app.honeybadger.io_, the generated URL will be the same.